tests/gtest/test_population_Index.cpp

```diff
@@ -22,16 +22,18 @@ namespace
 
         for (int fleet_ = 0; fleet_ < population.nfleets; fleet_++)
```

Comment:
Suggest: This test checks a single age in a single year, which is not an index. Add following comments to explain this: The test was developed to test the CalculateIndex function while the Integration test loops over all ages to test Index.